docs/your-questions-answered.md

```diff
@@ -0,0 +1,13 @@
+# Your Questions Answered
+
+This section of the Cloud Service Certification wiki answers questions asked by the CSC community.
+
+Ask your question by opening a [Support Issue](https://github.com/finos/cloud-service-certification/issues/new?template=Support_question.md) through GitHub.
+
+## Question
+How does Cloud Service Certification maintain the security of the environment and the protection of the environment for open source consumers? #18
+
+### Answer
+Cloud Service Certification has a peer review process to ensure the content created is functional (works as intended) and at least 2 members feel the content meets the control objective it was created for.This is OSS and it should not be trusted completely. 
```

Comment:
"2 members verify the content"

I think there's also a space after the "for." missing.

Reply:
Nicely caught. Thanks!